CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ project(tritontensorflowbackend LANGUAGES C CXX)
 #
 option(TRITON_ENABLE_GPU "Enable GPU support in backend." ON)
 option(TRITON_ENABLE_STATS "Include statistics collections in backend." ON)
+option(TRITON_JETSON_BUILD "Build for Jetson. " OFF)
+option(TRITON_TENSORFLOW_LICENSE_AVAILABLE "Whether the TensorFlow license is available in this build. " ON)
```

Comment:
This should not be an option. We should always include the license to comply with Apache 2.0.

Reply:
Please see the conversations here for the context of why this is here: https://github.com/triton-inference-server/tensorflow_backend/pull/89#discussion_r1163398695 and https://github.com/triton-inference-server/tensorflow_backend/pull/89#discussion_r1156538366.

I'm not sure what a good alternative would be. Since it's not included in the l4t containers as far as I know, we could probably clone it from elsewhere like GitHub if that's acceptable.